wemake_python_styleguide/visitors/ast/complexity/counts.py

```diff
@@ -72,6 +77,31 @@ def _check_decorators_count(self, node: ModuleMembers) -> None:
                 ),
             )
 
+    def _check_method_decorators(self, node: ModuleMembers) -> None:
+        parent_is_class = isinstance(get_parent(node), ast.ClassDef)
+
+        # Check whether filtered decorators are method decorators
+        is_method_decorator = []
```

Comment:
When something is named as `is_*` we expect it to be `bool`, not a list.
Let's call this variable `method_decorators`